### DISCUSSION TITLE
How do disable battery mode?

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#46] [Q&A] [answered]
Hi.

I am having hard time disabling the battery mode.
I have put these line in `config.el` and `early-config.el` but nothing works.

```lisp
(display-battery-mode -1)

(custom-set-variables
 '(display-battery-mode -1))
```

Thank you for `minemacs`.

**Comment by abougouffa** [comment, marked as answer]
Hello @azzamsa,

The `display-battery-mode` is enabled in a `+deffered!` block. If you add `(display-battery-mode -1)` directly in the `config.el` it will be called before the actual `display-battery-mode` is enabled.

So, adding this to your `config.el` should disable it.

```elisp
(+deferred!
 (display-battery-mode -1))
```
